tests/schema/fail/path-item-object-example.yaml

```
@@ -0,0 +1,64 @@
+openapi: 3.2.0
+info:
+  title: API
+  version: 1.0.0
+paths:
+  /pets/{id}:
+    get:
+      description: Returns pets based on ID
+      summary: Find pets by ID
+      operationId: getPetsById
+      responses:
+        '200':
+          description: pet response
+          content:
+            '*/*':
+              schema:
+                type: array
+                items:
+                  $ref: '#/components/schemas/Pet'
+        default:
+          description: error payload
+          content:
+            text/html:
+              schema:
+                $ref: '#/components/schemas/ErrorModel'
+    parameters:
+      - name: id
+        in: path
+        description: ID of pet to use
+        required: true
+        schema:
+          type: array
+          items:
+            type: string
+        style: simple
+    additionalOperations:
+      POST:
+        description: Returns pets based on ID
+        summary: Find pets by ID
+        operationId: postPetsById
+        request-body:
```

```suggestion
        requestBody:
```
`request-body` is not a fixed field of the Operation Object, which is why this file fails validation.

With correct spelling it passes validation because the schema only checks for "not post".

---

"le LLM m'a tuer" ;-)